shared/domains/chelonia/chelonia.js

```diff
@@ -1318,7 +1318,6 @@ export default (sbp('sbp/selectors/register', {
 
     let msg = SPMessage.createV1_0({
       contractID,
-      originatingContractID,
```

Comment:
Why is this line being deleted? Doesn't seem right...

Reply:
This is no longer used. Search the project for `originatingContractID`, and you'll notice that this isn't used as an SPMessage property.

Reply:
Are you sure? It does seem to be used when create a `'chelonia/out/keyRequest'` as part of `'gi.actions/group/join'`as well as with `'chelonia/out/actionEncrypted'` in `utils.js` as part of `encryptedAction`.

It's unclear to me though as `'chelonia/out/actionEncrypted'` doesn't seem to use it... however if it doesn't use it then the other code should be cleaned up too.

Reply:
Pretty sure, as `SPMessage` doesn't use this at all. `chelonia/out/keyRequest` has `originatingContractID` as a parameter, but it's _not_ part of the `SPMessage`. It used to be the case, but this was removed a while ago (I don't remember the specific PR or commit, but it's when the `'chelonia/contract/disconnect'` selector was added) as this information is now encrypted.

Reply:
> then the other code should be cleaned up too.

It should, but I think this is the only place that needs to be cleaned up. Other places may also need cleaning up, but just because `originatingContractID` is used it doesn't mean it needs to be cleaned up; it's just passing it to `SPMessage` that's no longer necessary nor used.